src/modules/always_on_top/dllmain.cpp

```diff
@@ -0,0 +1,272 @@
+#include "pch.h"
+#include <interface/powertoy_module_interface.h>
+#include <interface/powertoy_system_menu.h>
+#include <common/settings_objects.h>
+#include <set>
+#include "trace.h"
+#include "resource.h"
+
+extern "C" IMAGE_DOS_HEADER __ImageBase;
+
+BOOL APIENTRY DllMain(HMODULE hModule, DWORD  ul_reason_for_call, LPVOID lpReserved) {
+  switch (ul_reason_for_call) {
+  case DLL_PROCESS_ATTACH:
+    Trace::RegisterProvider();
+    break;
+  case DLL_THREAD_ATTACH:
+  case DLL_THREAD_DETACH:
+    break;
+  case DLL_PROCESS_DETACH:
+    Trace::UnregisterProvider();
+    break;
+  }
+  return TRUE;
+}
+
+const static wchar_t* MODULE_NAME = L"Always On Top";
+const static wchar_t* MODULE_DESC = L"Adds a command to the title bar context menu to set a window on top of all others.";
+const static wchar_t* HOTKEY_NAME = L"AlwaysOnTop_HotKey";
+const static wchar_t* HOTKEY_WINDOW_CLASS_NAME = L"HotkeyHandleWindowClass";
+const static wchar_t* OVERVIEW_LINK = L"https://github.com/microsoft/PowerToys/blob/master/src/modules/always_on_top/README.md";
+
+struct AlwaysOnTopSettings {
+  // Default hotkey WIN + ALT + T
```

Comment:
Let's use `Ctrl + Alt + T` as suggested by @bzoz since `Win + something` is very likely to cause compatibility issues with localized versions fo Windows.